metapackages/auto-instrumentations-web/tsconfig.esnext.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.esnext.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build/esnext",
+    "jsx": "react",
```

Comment:
Hmm probably not, I'll remove that.

Reply:
I've removed the [jsx](https://www.typescriptlang.org/tsconfig#jsx) option since there are no `.tsx` files in this package and added [skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck) to skip type checking of declaration files, matching the tsconfig for esm.